templates/search.html

```diff
@@ -6,8 +6,8 @@
     <div class="container">
         <div class="page-section__intro">
             <h2 id="browse-heading">Get to Know the Data</h2>
-            <p class="text--lead">Not sure exactly what you’re looking for? Start with a full set of data and then filter down to what you’re looking for.</p>
-            <p class="text--lead">We currently have data for all candidates, committees, and their reports and filings.</p>
+            <p class="text--lead">Want to learn more about federal election data but aren’t sure exactly what you’re looking for? Start with a full set of data, then apply specific filters to find the information most relevant to you.</p>
+            <p class="text--lead">OpenFEC currently has financial data for all candidates for federal office and all committees involved in federal races, and we will be adding more soon.</p>
```

Comment:
We don' have finical data for all types of committees. We do have their profile information. Though, maybe that is not a distinction that matters to most people.